data_contribution_2_data.md

```diff
@@ -0,0 +1,907 @@
+| local\_id\_type        | The type of participant id (local).            | String | This will always be local.                                     |
+| local\_participant\_id | Id that is used in PARTICIPANT_MANIFEST        | String |                                                                |
+| auth\_id\_type         | The type of participant id(chop)               | String | This will always be chop.                                      |
+| auth\_participant\_id  | Authorative Id of the participant. (Often MRN) | String | Use an 8 digit MRN. Left-pad the MRN with zeroes as necessary. |
+
+### manifests/participant\_family\_role
+
+The **participant\_family\_role.csv** file is only needed for some omics data. If you have family data (ie sequencing data from related family members), use this file to describe [relationships](https://github.research.chop.edu/arcus/rdm-project-template/blob/master/manifests/data_dicts/eHB_relationship_types_as_of_10_30.json). See below for an example.
+
+| local\_participant\_id | local\_relative\_id | relative\_family\_role |
+| -------------------- | ----------------- | -------------------- |
+| participant1         | participant2      | biological mother    |
+| participant2         | participant1      | biological son       |
+| participant1         | participant3      | biological father    |
+| participant3         | participant1      | biological son       |
+| participant1         | participant4      | biological sister    |
+| participant4         | participant1      | biological brother   |
+
+| column               | defintion                                                                                                             | type   |
+| -------------------- | --------------------------------------------------------------------------------------------------------------------- | ------ |
+| local\_participant\_id | The local id of a participant.                                                                                        | String |
+| local\_relative\_id    | The local id of a relative to the participant.                                                                        | String |
+| relative\_family\_role | The familial relationship of the relative to the participant. Use terms from eHB\_relationship\_types\_as_of\_10\_30.json. | String |
+
+### manifests/familyid_crosswalk
+
+- This manifest should be used with trio and cohort omics data. A trio will contain three participants, a cohort can contain hundreds. This file walks the name for the trio or cohort file with the local_participant_id's included in it.
+- This is also called a PED or pedigree file in bioinformatics workflows. A pedigree is a structured description of the familial relationships between samples, see this [link for more information](https://gatk.broadinstitute.org/hc/en-us/articles/360035531972-PED-Pedigree-format). 
+
```

Comment:
```suggestion

<!-- data-type="none" -->
```
Do you want the graph type "parallel crosswalk" to appear?  If not, I suggest you add the data type to the top of the table.

Reply:
We did not want a 'parallel crosswalk' and did not know how to get rid of it!